### ISSUE TITLE
ckeditor image upload error

### ISSUE THREAD

**Issue by henrysource** [sleeping-owl/admin#68]
As title. In ckeditor after i select image and hit send it to server. It shows
 
```
TokenMismatchException in compiled.php line 2486:
in compiled.php line 2486
at VerifyCsrfToken->handle(object(Request), object(Closure)) in VerifyCsrfToken.php line 17
at VerifyCsrfToken->handle(object(Request), object(Closure)) in compiled.php line 9019
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in compiled.php line 12171
at ShareErrorsFromSession->handle(object(Request), object(Closure)) in compiled.php line 9019
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in compiled.php line 10873
at StartSession->handle(object(Request), object(Closure)) in compiled.php line 9019
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in compiled.php line 11877
at AddQueuedCookiesToResponse->handle(object(Request), object(Closure)) in compiled.php line 9019
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in compiled.php line 11826
at EncryptCookies->handle(object(Request), object(Closure)) in compiled.php line 9019
at Pipeline->Illuminate\Pipeline\{closure}(object(Request)) in compiled.php line 2524
at CheckForMaintenanceMode->handle(object(Request), object(Closure)) in compiled.php line 9019
at Pipeline->Illuminate\Pipeline\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in compiled.php line 9010
at Pipeline->then(object(Closure)) in compiled.php line 1937
at Kernel->sendRequestThroughRouter(object(Request)) in compiled.php line 1924
at Kernel->handle(object(Request)) in index.php line 53
....
````
Is that need to setup session key same as host project? 
If so, where to setup?

**Comment by madeny** [comment]
I'm getting same thing here!

**Comment by madeny** [comment]
I just get it working again by " clearing config files and recreating cache.

**Comment by sleeping-owl** [comment]
This error will be fixed in the upcoming 3.0 version. For now you can disable csrf middleware for the `images/upload` route (http://www.camroncade.com/disable-csrf-for-specific-routes-laravel-5/).
